docs/ReleaseProcess.md

```diff
@@ -350,3 +308,99 @@ git pull
 
 [contributing-to-a-project]: https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines
 [chap]: https://github.com/input-output-hk/cardano-haskell-packages
+
+# The first time we had to break this release process
```

Comment:
It's great we record this, however, we might need to create another piece of documentation for the people that just want to either cut a release or understand our release process. If you think this makes sense we could create a new issue for this.